configmap/parse_test.go

```diff
@@ -19,15 +19,19 @@ package configmap
 import (
 	"testing"
 	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"k8s.io/apimachinery/pkg/util/sets"
 )
 
 type testConfig struct {
-	str string
-	boo bool
-	i32 int32
-	i64 int64
-	f64 float64
-	dur time.Duration
+	Str string
```

Comment:
Why do these fields need to be exported?

Reply:
to make `cmp.Equal` work.

Reply:
https://godoc.org/github.com/google/go-cmp/cmp#AllowUnexported

Reply:
Oh neat, I thought there was only an option to ignore unexported!

Reply:
fixed!